ui/ops/src/util/array.js

```diff
@@ -0,0 +1,66 @@
+import {nextTick} from 'vue';
+
+/**
+ * Iterates over large arrays asynchronously
+ *
+ * @param {Array} array
+ * @param {function} cb
+ * @param {boolean} batched
+ */
+export async function iterateLargeArray(array, cb, batched = false) {
```

Comment:
I think the function name could do with a little work. There's no mention that it calls Vue.nextTick, it's not clear what large arrays are, etc. I'd be tempted to say that this doesn't need to exist at all with the new batch function.

```js
arr.forEach(fn);
// becomes
batch(arr, 10).forEach((b) => {
  b.forEach(fn);
})
// or with async
for (const b of batch(arr, 10)) {
  b.forEach(fn);
  await nextTick();
}
```

Which I think is easy to understand without the need for the iterator func.